.github/workflows/doc-build.yml

```diff
@@ -195,11 +203,11 @@ jobs:
         if: (success() || failure()) && steps.buildlivedoc.outcome == 'success'
         run: |
           mkdir -p ./livedoc
-          cp -r -L /sage/local/share/doc/sage/html ./livedoc
-          cp -r -L /sage/local/share/doc/sage/pdf ./livedoc
-          cp  /sage/local/share/doc/sage/index.html ./livedoc
+          # We copy everything to a local folder
+          docker cp BUILD:/sage/local/share/doc/sage/html livedoc
+          docker cp BUILD:/sage/local/share/doc/sage/pdf livedoc
+          docker cp BUILD:/sage/local/share/doc/sage/index.html livedoc
           zip -r livedoc.zip livedoc
-        shell: sh .ci/docker-exec-script.sh BUILD . {0}
 
```

Comment:
Am I doing things right here?

Reply:
I am asking because there is `-L` previously, but `docker cp` does not have the flag.

Reply:
For the corresponding option for `docker cp`, see https://github.com/sagemath/sage/pull/36730/files

Reply:
But the right way to do this, I think, is to finally do https://github.com/sagemath/sage/issues/35538#issuecomment-1532477914 as part of the docbuild...

Reply:
Thanks. I use `-L` equivalent to `--follow-link` for now.

Reply:
Rather more explicit `--follow-link` as in #36730.